.github/templates/linux_ci_workflow.yml.in

```diff
@@ -243,6 +243,8 @@ jobs:
   render_test_results:
     if: always()
     needs:
+      - calculate-docker-image
+      - build
```

Comment:
Isn't there an intrinsic dependency within this? Like if you depend on `test` which depends on `build` and `calculate-docker-image` wouldn't it just auto depend on the dependencies of its dependencies?

Reply:
yeah exactly; see my comments above

Reply:
fixed.